.clang-format

```diff
@@ -1,16 +1,26 @@
 ---
-BasedOnStyle: Google
-Language: Cpp
-Standard: Cpp03
+# Indentation for all files
+ColumnLimit: 120
+IndentWidth: 4
 TabWidth: 4
 UseTab: Never
-ColumnLimit: 120
-NamespaceIndentation: All
+
+# C++ Options
+Language: Cpp
+Standard: c++17
```

Comment:
I'd remove and leave it as default, which is `Auto` and let's `clang-format` detect the standard